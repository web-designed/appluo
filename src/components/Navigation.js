import React from 'react'
import { NavLink } from 'react-router-dom'

const Navigation = () => {
   return (
      <nav class="navbar navbar-expand-sm bg-dark">
         <ul class="nav nav-pills mb-0">
            <li class="nav-item"><NavLink activeClassName="active" className="text-white nav-link" to="/" exact={true}>Dashboard</NavLink></li>
            <li class="nav-item"><NavLink activeClassName="active" className="text-white nav-link" to="/add">Add Cleaning</NavLink></li>
            <li class="nav-item"><NavLink activeClassName="active" className="text-white nav-link" to="/stats">Chart</NavLink></li>
         </ul>
      </nav>
   )
}

export default Navigation