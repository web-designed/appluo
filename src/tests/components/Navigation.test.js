import React from 'react'
import { shallow } from 'enzyme'
import Navigation from '../../components/Navigation'

test('should render Navigation correctly', () => {
   const wrapper = shallow(<Navigation />)
   expect(wrapper).toMatchSnapshot()
})