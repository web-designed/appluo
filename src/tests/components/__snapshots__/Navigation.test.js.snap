// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Navigation correctly 1`] = `
<nav
  class="navbar navbar-expand-sm bg-dark"
>
  <ul
    class="nav nav-pills mb-0"
  >
    <li
      class="nav-item"
    >
      <NavLink
        activeClassName="active"
        aria-current="page"
        className="text-white nav-link"
        exact={true}
        to="/"
      >
        Dashboard
      </NavLink>
    </li>
    <li
      class="nav-item"
    >
      <NavLink
        activeClassName="active"
        aria-current="page"
        className="text-white nav-link"
        to="/add"
      >
        Add Cleaning
      </NavLink>
    </li>
    <li
      class="nav-item"
    >
      <NavLink
        activeClassName="active"
        aria-current="page"
        className="text-white nav-link"
        to="/stats"
      >
        Chart
      </NavLink>
    </li>
  </ul>
</nav>
`;
